sushy/interop_profiles/ServiceRoot.v1_0_0.json

```diff
@@ -0,0 +1,73 @@
+{
+	"SchemaDefinition": "RedfishInteroperabilityProfile.v1_0_0",
+	"ProfileName": "RedfishProfile",
+	"ProfileVersion": "1.0.0",
+	"Purpose": "Redfish Interoperability profile",
+	"OwningEntity": "Dell",
```

Comment:
This should be `Dell Inc.`, instead of `Dell`.

Reply:
If the intent is to contribute this back to OpenStack, then it might need to say the owning entity is OpenStack. Property values can't easily span multiple lines in JSON, so I would recommend avoiding that.

Reply:
Hey @mraineri, I believe we can claim ownership of the copyright and contribute it to OpenStack. Please see [this example](https://opendev.org/openstack/ironic/src/commit/c871622ff589c3e5eb21f5938c7e07db7c9765f3/ironic/drivers/modules/drac/management.py#L5-L17) from our ironic iDRAC driver code.

How could we do something similar in a JSON Redfish Interoperability Profile?

Reply:
I think that's largely a process you need to work out with OpenStack; I know when people throw stuff over the wall to the DMTF, it usually has the originator's copyright info, and then that gets replaced, or added with DMTF info.

Reply:
Also, you can add a copyright notice. On line 2 of the file, you can add `"@Redfish.Copyright": "Dell Inc 2020 <blah blah blah>"`

Reply:
Beneath that you can add a license term like: `"@Redfish.License": "<Whatever license you want to reference>"`

Reply:
Thank you, @mraineri!

@spranjali, let us add two (2) lines below the first line of the profile, which is an opening curly brace.

    {
        "@Redfish.Copyright": "Copyright (c) 2020 Dell Inc. or its subsidiaries."
        "@Redfish.License": "Apache License, Version 2.0. For full text, see link: http://www.apache.org/licenses/LICENSE-2.0"
        <snip>

Reply:
Added in the latest commit